contrib/tealdeer/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "tealdeer"
+pkgver = "1.6.1"
+pkgrel = 0
+build_style = "cargo"
```

Comment:
should also use `--features native-tls` and use openssl-devel

Reply:
ah, that's only in master

Reply:
yeah that's a blocker, need to patch that then

Reply:
you'll need to add a

```
def do_prepare(self):
    # we patch the lockfile so vendor after patch
    pass


def post_patch(self):
    from cbuild.util import cargo

    self.cargo.vendor()
    cargo.setup_vendor(self)
```

and then generate a patch of the changed lock (`cargo update -p reqwest --precise=0.11.12`, or just update reqwest entirely) after adding the feature. but actually just that alone won't work, because by default it also uses `
native-roots = ["reqwest/rustls-tls-native-roots"]`, which will build ring anyway (and then not use it), so the build will fail. so you'd also have to remove those, and then also patch out the assertion in main.rs that checks for the feature... (it does work in the end)

Reply:
> and then generate a patch of the changed lock   

(and cargo.toml, and the changes in main.rs)